### DISCUSSION TITLE
Is @babel/runtime deprecated?

### DISCUSSION THREAD

**Discussion by 2239559319** [babel/babel#13546] [Q&A]
In @babel/plugin-transform-runtime, use In the @babel/plugin-transform-runtime option, the file packaged with corejs2 is loaded @babel/runtime-corejs2, and the file packaged with corejs3 is loaded @babel/runtime-corejs3. Where is @babel/runtime?

**Comment by nicolo-ribaudo** [comment]
If you don't specify the `corejs` option, it uses `@babel/runtime`.

**Comment by 2239559319** [comment]
> In @babel/plugin-transform-runtime, use In the @babel/plugin-transform-runtime option, the file packaged with corejs2 is loaded @babel/runtime-corejs2, and the file packaged with corejs3 is loaded @babel/runtime-corejs3. Where is @babel/runtime?

And I switch corejs to false then @babel/ plugin-tranform-runtime is not working.

**Reply by nicolo-ribaudo** [reply]
"not working" is a bit vague to understand what's happening. Is it throwing? Is it just "doing nothing"?

**Reply by 2239559319** [reply]
> "not working" is a bit vague to understand what's happening. Is it throwing? Is it just "doing nothing"?

do nothing

**Reply by 2239559319** [reply]
> "not working" is a bit vague to understand what's happening. Is it throwing? Is it just "doing nothing"?

![image](https://user-images.githubusercontent.com/34960995/125169866-c4762b00-e1de-11eb-9bc9-f5be934a2e56.png)
![image](https://user-images.githubusercontent.com/34960995/125169876-cc35cf80-e1de-11eb-9feb-44d69ed15b86.png)
![image](https://user-images.githubusercontent.com/34960995/125169883-d48e0a80-e1de-11eb-8e76-12b1ada11154.png)
Here are my files

**Reply by 2239559319** [reply]
> "not working" is a bit vague to understand what's happening. Is it throwing? Is it just "doing nothing"?


I found out only because @babel/transform-runtime does not polyfilled Map

**Reply by nicolo-ribaudo** [reply]
It's because `@babel/runtime` by default only extracts helpers: if you need polyfills, you must use the `corejs` option.

In your case it's doing nothing because that code doesn't need any runtime helper.

**Comment by 2239559319** [comment]
> If you don't specify the `corejs` option, it uses `@babel/runtime`.

I use plugin without options and the plugin is not working.
